templates/stream.html

```diff
@@ -1,17 +1,23 @@
 {% extends "base.html" %} {% block content %}
 
-<h1>socialDISTRIBUTION</h1>
 <div class="container">
-	<a href="{% url 'posts:new' %}">New post</a>
-
+	<ul>
+		<li><a href="{% url 'posts:new' %}">New post</a></li>
+		<li><a href="Friend Requests">Friend Requests</a></li>
+	</ul>
+	
+	
+</div>
+<div class="post">
```

Comment:
I think you want to put this in as the outermost wrapper in `_stream_post.html` if you want to style individual posts (i.e. card like component)

Reply:
Removing this because we're not using it yet